lib/stripe/bank_account.rb

```diff
@@ -0,0 +1,19 @@
+module Stripe
+  class BankAccount < APIResource
+    include Stripe::APIOperations::Update
+    include Stripe::APIOperations::Delete
+    include Stripe::APIOperations::List
+
+    def url
+      if respond_to?(:customer)
+        "#{Customer.url}/#{CGI.escape(customer)}/sources/#{CGI.escape(id)}"
```

Comment:
Yup, they are. For the 1 LoC may as well add support to the bindings. Attempts to use it without access to the beta will cause an error.